cluster-provision/common-scripts/config-cni.sh

```diff
@@ -1,21 +1,16 @@
-#! /bin/bash
+#!/bin/env bash
 
 set -e
 
-function configure_calico(){
-    cat <<EOF >/etc/NetworkManager/conf.d/calico.conf
-[keyfile]
-unmanaged-devices=interface-name:cali*;interface-name:tunl*
-EOF
-
-    sysctl -w net.netfilter.nf_conntrack_max=1000000
-    echo "net.netfilter.nf_conntrack_max=1000000" >> /etc/sysctl.conf
-
-    systemctl restart NetworkManager
-}
+cni_name=$1
+script_d=$cni_name/script.d
+default_cidr="192.168.0.0/16"
+pod_cidr="10.244.0.0/16"
 
 function configure_cni(){
-    if [[ $1 =~ calico ]]; then
-        configure_calico
+    if [[ -d $script_d ]]; then
+        for s in $(ls script_d); do
```

Comment:
How about:
```
cni_config_scripts=$(ls ...)
for script in $cni_config_scripts
 ./$script ..
.
.
```

Reply:
If we do this and we don't have the scripts.d the script is going to fail